charts/activiti-cloud-full-example/values.yaml

```diff
@@ -440,8 +440,8 @@ activiti-modeling-app:
   nameOverride: activiti-modeling-app
   image:
     repository: activiti/activiti-modeling-app
-    tag: master
-    pullPolicy: Always
+    tag: 7.9.0
+    pullPolicy: IfNotPresent
```

Comment:
since it's been discontinued, why do we need to keep this reference at all?

Reply:
Even if the project has been discontinued, I think it still make sense to keep a reference to the latest released version of it, so that one can still have an easy access to a BPMN Modeler.

Reply:
The whole point of discontinuing was to not maintain it: I don't see the point of keeping a reference to a release that might get unaligned, but up to you.

Reply:
I agree with Anahide. We risk to forget about this reference causing misunderstanding and confuse stakeholder.

Reply:
Ok, I see your point and I need to agree that if the backend evolves in the way where it's no longer compatible with the frontend we'll not be able to fix it. I'll update the PR to remove all the references to Modeling App.